### PR TITLE
add dockerfile

### PR DESCRIPTION
**Pull request by houqp**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.10
+WORKDIR /go/src/github.com/teralytics/prometheus-ecs-discovery/
+COPY main.go .
+RUN go get .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o app .
```
Comment (vincepii) [inline Dockerfile]: Can you replace `app` with `prometheus-ecs-discovery`?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+FROM alpine:latest
+RUN apk --no-cache add ca-certificates
+WORKDIR /root/
+COPY --from=0 /go/src/github.com/teralytics/prometheus-ecs-discovery/app .
```
Comment (vincepii) [inline Dockerfile]: Can you replace `app` with `prometheus-ecs-discovery`?

**Comment by houqp** [comment]
Sure, updated :)

**Comment by vincepii** [comment]
> Sure, updated :)

Thanks!

**Pull request merged**